src/pyforest/user_specific_imports.py

```diff
@@ -60,9 +60,24 @@ def _get_imports_from_user_settings(user_imports_path) -> list:
 
 
 def _assign_imports_to_globals(import_statements: list, globals_) -> None:
-    symbols = [import_statement.split()[-1] for import_statement in import_statements]
-
-    for symbol, import_statement in zip(symbols, import_statements):
+    symbols = []; new_import_statements = []
+    
+    for import_statement in import_statements:
+        def process(statement):
+            symbols.append(statement.split()[-1])
+            new_import_statements.append(statement)
+
+        if "," not in import_statement:
+            process(import_statement)
+        else:
```

Comment:
Thanks for reply!

My PR doesn't affect any original behaviors.

PR uses new line 70. to judge whether the import statement includes multiple imports at first.

if "," not in import_statement,
"from keras.layers import LSTM" still behaves as original code
like original line 63, which is similar with new line 66~68.

if "," is in import_statement, "from keras.layers import LSTM, Dropout, Dense",
it will become
"from keras.layers import LSTM"
"from keras.layers import Dropout"
"from keras.layers import Dense" with new line 73~78.

Then, the following behaviors are still the same as the original line 63.

In the other words,
I extend the original line 63 into new line 63~78 for the judgement and split.